Codecademy/gamut
# feat(icons): add levels icon

### Overview


Adds `LevelIcon` to `gamut-icons`

Stroke of 1.5 was too much, got confirmation from our designer to reduce it to 1
Setting the viewBox to 0 0 24 24 caused the icon to be too small, using the default viewBox numbers from streamline



### PR Checklist

- [x] Related to designs:https://www.figma.com/file/XSeoAT1Q97FBZc1uju2Cgx/Course-Landing-Page-v1?node-id=1109%3A149863

#### Testing Instructions
Confirm the icon shows up in storybook



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |
